bin/server_package_standalone

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+###########################################################################
+# Packaging script which creates standalone package.
+#
+# Usage:
+#> bin/server_package_standalone
+
+set -e
+
+BINARY=${BINARY:="build/server/mysterium_server"}
```

Comment:
What does this syntax mean?

Reply:
Variable can be passed to script via environment variables, but has default value